evm/vm/forks/frontier/computation.py

```diff
@@ -14,8 +18,26 @@
     keccak,
 )
 
+from .opcodes import FRONTIER_OPCODES
+
+FRONTIER_PRECOMPILES = {
+    force_bytes_to_address(b'\x01'): precompiles.ecrecover,
+    force_bytes_to_address(b'\x02'): precompiles.sha256,
+    force_bytes_to_address(b'\x03'): precompiles.ripemd160,
+    force_bytes_to_address(b'\x04'): precompiles.identity,
+}
+
 
 class FrontierComputation(BaseComputation):
+    def __init__(self, vm_state, message):
+        super(FrontierComputation, self).__init__(
+            vm_state,
+            message,
+        )
+        # Override
+        self.opcodes = FRONTIER_OPCODES
```

Comment:
Same suggestion here to remove these from `__init__` and set them as class properties.